proxy.go

```diff
@@ -451,12 +452,18 @@ func (rp *reverseProxy) serveFromCache(s *scope, srw *statResponseWriter, req *h
 			tmpFileRespWriter.WriteHeader(srw.statusCode)
 		}
 
-		errString, err := toString(reader)
-		if err != nil {
-			log.Errorf("%s failed to get error reason: %s", s, err.Error())
+		var errReason string
```

Comment:
Can you add a default value at "" in order to avoid a nil errReason? It could lead to future bugs

Reply:
added default value